app/views/api/v0/classified_listings/show.json.jbuilder

```diff
@@ -0,0 +1,34 @@
+json.type_of              "classified_listing"
+json.id                   @classified_listing.id
+json.title                @classified_listing.title
+json.slug                 @classified_listing.slug
+json.body_markdown        @classified_listing.body_markdown
+json.bumped_at            @classified_listing.bumped_at
+json.cached_tag_list      @classified_listing.cached_tag_list
+json.category             @classified_listing.category
+json.contact_via_connect  @classified_listing.contact_via_connect
+json.processed_html       @classified_listing.processed_html
+json.published            @classified_listing.published
+json.slug                 @classified_listing.slug
+json.last_buffered        @classified_listing.last_buffered
+json.tag_list             @classified_listing.tag_list
+
+json.user do
```

Comment:
see the notes on the index json builder

BTW if the show and the index are the same, inside the loop of the index you can just render a common partial